gkr_iop/src/circuit_builder.rs

```diff
@@ -888,6 +888,38 @@ impl<'a, E: ExtensionField> CircuitBuilder<'a, E> {
         )
     }
 
+    pub fn assert_double_u8<NR, N>(
+        &mut self,
+        name_fn: N,
+        a_expr: Expression<E>,
+        b_expr: Expression<E>,
+    ) -> Result<(), CircuitBuilderError>
+    where
+        NR: Into<String>,
+        N: FnOnce() -> NR,
+    {
+        self.namespace(
+            || "assert_double_u8",
+            |cb| cb.lk_record(name_fn, LookupTable::DoubleU8, vec![a_expr, b_expr]),
+        )
+    }
+
+    pub fn assert_double_ux<NR, N, const C: usize>(
```

Comment:
we can remove this function and just use `assert_double_u8`

Reply:
fixed in https://github.com/scroll-tech/ceno/pull/1044/commits/ef35883ecf4fe51575d9434f5969e2cfe98a2741 :)